### DISCUSSION TITLE
Possible to not send a note?

### DISCUSSION THREAD

**Discussion by ar06x** [melanchall/drywetmidi#126] [Q&A] [answered]
Is it possible to stop the note from sending in `NoteCallback` or there some other way?

**Comment by melanchall** [comment, marked as answer]
Yes, please see [documetation on NoteCallback](https://melanchall.github.io/drywetmidi/api/Melanchall.DryWetMidi.Devices.NoteCallback.html):

> Returns
> ...
> Data of the new note which is rawNoteData processed by the callback; **or null if note should be ignored**.

So just return `null` in a callback to skip a note.
